lib/minimart/mirror/inventory_builder.rb

```diff
@@ -33,10 +34,16 @@ def build!
       # precedence over information found elsewhere.
       def install_cookbooks_with_explicit_location
         inventory_requirements.each_with_explicit_location do |requirement|
-          requirement.fetch_cookbook do |cookbook|
-            validate_cookbook_against_local_store(cookbook, requirement)
-            add_artifact_to_graph(cookbook)
-            add_cookbook_to_local_store(cookbook.path, requirement.to_hash)
+          begin
```

Comment:
Hey @tapickell ,
I've included a full stacktrace as a comment in the PR, but the failure happens pretty deep in the internals of minimart. If we catch and ignore it in `lib/minimart/cookbook.rb:19` (where it's actually thrown) then the `validate_cookbook_against_local_store`, `add_artifact_to_graph`, etc methods will still attempt to run, throwing additional errors.